### ISSUE TITLE
job template spec.networks not returning an IP

### ISSUE THREAD

**Issue by drnic** [cloudfoundry/bosh-init#18]
Hopefully I'm describing the issue correctly.

I discovered that this template when deployed by bosh-init was converting:

```ruby
-forwardHost=<%= my_ip %>
```

into

```
-forwardHost=
```

see - https://github.com/concourse/concourse/blob/master/jobs/tsa/templates/tsa_ctl.erb#L66

https://github.com/concourse/concourse/blob/master/jobs/tsa/templates/tsa_ctl.erb#L15-L34 in `discover_external_ip` it finds the first `spec.networks` and returns `network.ip` as ""

Notable it doesn't `raise` an error - https://github.com/concourse/concourse/blob/master/jobs/tsa/templates/tsa_ctl.erb#L27

**Comment by cppforlife** [comment]
Yup, currently bosh-init does not properly populate network info before rendering templates.
